### PR TITLE
Restructure and refactor helpers

### PR DESCRIPTION
**Pull request by RonaldDijkstra**
- [x] Move helpers to specific helper files
- [x] Remove unneeded helpers
- [x] Refactor helpers
  - [x] page_classes
  - [x] canonical_tag
  - [x] country_flags eg language_switcher
  - [x] page_title 
  - [x] alternate_link_tags
  - [x] meta_description
  - [x] default_path
  - [x] nav_link_to

### CODE REVIEW

**Comment by RonaldDijkstra** [comment]
Only the Metrics comments left. 🍻

File: config.rb [hunk 1]
```diff
@@ -366,299 +368,6 @@ def call(exception, locale, key, options)
...
-          hash[key] = false
-          puts "\033[31mWARNING: Download link does not exist '#{value}'\033[0m"
-        end
-      end
-    end
-  end
-end
+# helpers do
+#
+# end
```
Comment (fatboypunk) [inline config.rb]: this can be removed

Comment (RonaldDijkstra) [reply config.rb]: Sure

File: helpers/language_switcher.rb [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+  def full_jobs_url(lang)
+    lang == :en ? "/" : full_url("/jobs/", lang)
+  end
+
+  # Define full root url
+  def full_root_url(lang)
+    full_url("", lang)
+  end
+
+  # Is 404 page?
```
Comment (RonaldDijkstra) [inline helpers/language_switcher.rb]: #1464

**Comment by RonaldDijkstra** [comment]
https://github.com/thoughtbot/middleman-aria_current/blob/master/lib/middleman-aria_current/extension.rb

**Comment by RonaldDijkstra** [comment]
Okay, now this all works I'll chop it up in different branches, so it's a bit better reviewable

**Comment by RonaldDijkstra** [comment]
Bye

**Pull request closed without merging**